ComponentKit/LayoutComponents/CKStackPositionedLayout.mm

```diff
@@ -43,11 +43,23 @@ static CKStackPositionedLayout stackedLayout(const CKStackLayoutComponentStyle &
   const auto largestChildCrossSize = it == unpositionedLayout.items.end() ? 0 : crossDimension(style.direction, it->layout.size);
   const auto minCrossSize = crossDimension(style.direction, constrainedSize.min);
   const auto maxCrossSize = crossDimension(style.direction, constrainedSize.max);
+  const auto maxStackSize = stackDimension(style.direction, constrainedSize.max);
+
   const CGFloat crossSize = MIN(MAX(minCrossSize, largestChildCrossSize), maxCrossSize);
 
+  CGFloat maxLineHieght = 0;
```

Comment:
Also, line height I don't think is a very good name. This algorithm is built to work in either horizontal or vertical directions, so I doubt this is a height in every case right? Height implies vertical directionality of the value.